examples/typescript-react/js/app.tsx

```diff
@@ -0,0 +1,181 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
```

Comment:
`trailing` and `white` are [deprecated](https://github.com/jshint/jshint/releases/tag/2.5.0).